### ISSUE TITLE
Clairctl cannot pull from docker registry

### ISSUE THREAD

**Issue by jdel** [jgsqware/clairctl#3]
I am facing an issue trying to pull from the Docker hub with clairctl:

```
sf-clair ~ # clairctl --log-level debug pull jgsqware/registry-ui:latest
DEBU[0000] Using config file: /root/clairctl.yml
client quit unexpectedly
FATA[0004] retrieving manifest for "jgsqware/registry-ui:latest": Received error when trying to fetch the specified tag: it might not exist or you do not have access
```

This used to work with hyperclair. 

I am using the following config:

```
sf-clair ~ # clairctl health

Clair: ✔
sf-clair ~ # clairctl version

Clairctl version
sf-clair ~ # cat clairctl.yml
clair:
  port: 6060
  healthPort: 6061
  uri: http://clair
  priority: Low
  report:
    path: /var/opt/data/hyperclair/reports
    format: html
clairctl:
  tempFolder: /var/opt/tmp/hyperclair
```

**Comment by jgsqware** [comment]
Got the same locally. I will investigate it

**Comment by jdel** [comment]
I will do some testing and create a few more issues, don't freak out :)

**Comment by jgsqware** [comment]
Ok found why.
In Clairctl, I use the `docker/distribution` package that has changed since last modification (few weeks ago) and docker hub api return manifestv2 now and I currently support only v1.

So I need some development to fix it.

However, Thanks a lot for the deep testing and issu creation! really appreciate it

**Comment by jdel** [comment]
No problem, I'd really like to have clairctl working to integrate with our CI 😆.
Technically, my first experience with golang was the cross compiling done yesterday and this morning, so unfortunately I cannot help much with the code.

**Comment by jgsqware** [comment]
Ok I've managed to fix the pull you can follow it in this branch: [fix-pull](https://github.com/jgsqware/clairctl/tree/fix-pull)

**Comment by jdel** [comment]
That works for me, and it seems to fix #5 too.

**Comment by jgsqware** [comment]
closed by #10
